### DISCUSSION TITLE
prevent black-full screen from appearing when playing an audio inside an iframe

### DISCUSSION THREAD

**Discussion by joostwmd** [react-native-community/discussions-and-proposals#711] [Ideas]
hello community, 

first of all I hope this is the right place to bringt this up. I want to embed an iFrame from spotify. That works well, but when playing an audio snippet form that frame, a black full screen with audio controls appears. Is there a way to prevent this behavior?

Here is an exmaple of the iFrame that i want to embed for clarity:

```
<iframe style="border-radius:12px" src="https://open.spotify.com/embed/artist/4wbbDaTxjWd2fcghJQHIgn?utm_source=generator" width="100%" height="300" frameBorder="0" allowfullscreen="" allow="autoplay; clipboard-write; encrypted-media; fullscreen; picture-in-picture" loading="lazy"></iframe>
```

**Comment by kelset** [comment]
👋 there - as stated in the README and the issue template, this repo is for long-form discussions. Yours seems more a question for [StackOverflow instead](https://stackoverflow.com/questions/tagged/react-native).
